### ISSUE TITLE
depth doesn't work? Wrong chart location is being used, returns 404

### ISSUE THREAD

**Issue by jackivanov** [helm/chartmuseum#433]
I've been following multitenancy [example](https://github.com/helm/chartmuseum#multitenancy), but couldn't get it up and running

**Steps to reproduce:**

Chart tgz /tmp/charts/nginx/nginx-ingress-0.8.0.tgz

```
% tree /tmp/charts
/tmp/charts
`-- nginx
    `-- nginx-ingress-0.8.0.tgz

1 directory, 1 file
```

Run the server:
```
chartmuseum --debug --port=8080 --storage="local" --storage-local-rootdir=/tmp/charts --depth=1
```

Try to add a repo:
```
% helm repo add nginx http://localhost:8080/nginx
Error: looks like "http://localhost:8080/nginx" is not a valid chart repository or cannot be reached: failed to fetch http://localhost:8080/nginx/index.yaml : 500 Internal Server Error
```

Try to pull the index:
```
% curl http://localhost:8080/nginx/index.yaml
{"error":"backend storage: chart not found: \"open /tmp/charts/nginx-ingress-0.8.0.tgz: no such file or directory\""}
```

**/tmp/charts/nginx-ingress-0.8.0.tgz** is a wrong location, it should be **/tmp/charts/nginx/nginx-ingress-0.8.0.tgz**

```
2021-03-11T18:06:52.880Z	DEBUG	Starting internal event listener
2021-03-11T18:06:52.880Z	INFO	Starting ChartMuseum	{"host": "0.0.0.0", "port": 8080}

2021-03-11T18:07:03.180Z	DEBUG	[1] Incoming request: /nginx/index.yaml	{"reqID": "3fbe7eee-3231-4f65-ae26-9cb743ff240f"}
2021-03-11T18:07:03.180Z	DEBUG	[1] Fetching chart list from storage	{"repo": "nginx", "reqID": "3fbe7eee-3231-4f65-ae26-9cb743ff240f"}
2021-03-11T18:07:03.181Z	ERROR	[1] backend storage: chart not found: "open /tmp/charts/nginx-ingress-0.8.0.tgz: no such file or directory"	{"repo": "nginx", "reqID": "3fbe7eee-3231-4f65-ae26-9cb743ff240f"}
2021-03-11T18:07:03.181Z	ERROR	[1] Request served	{"path": "/nginx/index.yaml", "comment": "", "clientIP": "::1", "method": "GET", "statusCode": 500, "latency": "883.551µs", "reqID": "3fbe7eee-3231-4f65-ae26-9cb743ff240f"}
2021-03-11T18:07:14.740Z	DEBUG	[2] Incoming request: /nginx/index.yaml	{"reqID": "d161360d-c542-497d-afbe-368ddab0195f"}
2021-03-11T18:07:14.740Z	DEBUG	[2] Entry found in cache store	{"repo": "nginx", "reqID": "d161360d-c542-497d-afbe-368ddab0195f"}
2021-03-11T18:07:14.740Z	DEBUG	[2] Fetching chart list from storage	{"repo": "nginx", "reqID": "d161360d-c542-497d-afbe-368ddab0195f"}
2021-03-11T18:07:14.740Z	ERROR	[2] backend storage: chart not found: "open /tmp/charts/nginx-ingress-0.8.0.tgz: no such file or directory"	{"repo": "nginx", "reqID": "d161360d-c542-497d-afbe-368ddab0195f"}
2021-03-11T18:07:14.740Z	ERROR	[2] Request served	{"path": "/nginx/index.yaml", "comment": "", "clientIP": "::1", "method": "GET", "statusCode": 500, "latency": "187.539µs", "reqID": "d161360d-c542-497d-afbe-368ddab0195f"}
```

**Comment by jdolitsky** [comment]
@jackivanov - are you using 0.13? This bug appears related: https://github.com/helm/chartmuseum/pull/423

**Comment by jackivanov** [comment]
@jdolitsky ah, yes, sorry, forgot to specify the version

v0.13.0 (build 3a24659)

**Comment by jdolitsky** [comment]
If youre able to build from `main` branch and let us know if that fixes, or use `0.12`. We are working on getting out a `0.13.1` release

**Comment by jackivanov** [comment]
@jdolitsky v0.13.1 (build 79bb39c) works fine, thanks! I'm closing the issue, please bump the fish-food pins
